mica/archive/obsid_archive.py

```diff
@@ -39,9 +39,10 @@ def parse_obspar(file):
                'r': float,
                's': str}
     try:
-        lines = gzip.open(file).readlines()
+        lines = gzip.open(file, 'rb').readlines()
```

Comment:
You should be able to put the `encoding` and `errors` in the calls to `gzip.open` and `open`.

Reply:
Sure.  We had just had those comments about what is in scope.  This also applies to the use of context managers.

Reply:
I think of this as working on an old house, so any wiring or plumbing that gets touched should be brought up to code.  As long as there are no fires burning we can take the time to do things right, reinforcing good practices and making incremental improvements.

Reply:
Sure.  I just wasn't sure about level of fires burning.  The ingest is currently broken which actually makes it harder to do things like V&V, and I wasn't sure if I'd catch load review this week.  I'm feeling slow.

Also, if we need LR approval for all ska3 code, we should have broken mica up into a separate project for each of these tasks because I still figure I'm going to bring this for review and then notice a tiny issue when run in flight that will keep everything from working.

Reply:
We can get this (and subsequent bug fixes) quickly approved out-of-cycle if need be.